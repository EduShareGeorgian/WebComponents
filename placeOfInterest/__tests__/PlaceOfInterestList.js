import renderer from "react-test-renderer"
import React from "react"
import {DetailsList} from "office-ui-fabric-react"

describe( "Place of Interest list component", () => {
  it ("should render itself with an Office Fabric DetailsList component", () => {
    const renderedComponent = renderer.create(
        <DetailsList></DetailsList>
    ).toJSON()
    expect(renderedComponent).toMatchSnapshot()
  })
})