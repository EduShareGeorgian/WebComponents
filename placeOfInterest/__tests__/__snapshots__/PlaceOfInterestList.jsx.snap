// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Place of Interest list component without items should render an empty div (React DOM snapshot method) 1`] = `<div />`;
